std_msgs/Header header
string vehicle_id
string status
float64 battery_level
geometry_msgs/Pose pose
geometry_msgs/Twist velocity
string[] active_systems 